src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -90,6 +92,12 @@ public ProAI(final String name, final String type) {
     storedPurchaseTerritories = null;
     storedPoliticalActions = null;
     storedStrafingTerritories = new ArrayList<Territory>();
+    initializeBattleCalculator();
+  }
+
+  public void initializeBattleCalculator() {
```

Comment:
Please convert this to be private. A class is responsible for its own initialization generally. Breaks encapsulation by having other classes manage the state of this class.